DevCycle/DVCClient.swift

```diff
@@ -164,8 +164,8 @@ public class DVCClient {
         }
     }
     
-    func setEnvironmentKey(_ environmentKey: String) {
```

Comment:
@jsalaber Thoughts on changing the main builder function here from `setEnvironmentKey()` to `setSDKKey()`. This would basically be a breaking change for everyone who has implemented the iOS SDK so far. We could have a `setEnvironmentKey()` and `setSDKKey()` and set the `setEnvironmentKey()` to be depricated...

Reply:
yeah i don't like the idea of removing it and have it as a breaking change for just a simple rename, we can keep both and mark the other as deprecated.